src/docs/dump-formats.dox

```diff
@@ -97,6 +99,9 @@ literal character is written as a pair of characters (first the
 high-nibble and then the low-nibble).  For example, "0a" would be an
 ASCII newline character and "1b" would be an ASCII escape character.
 
+The printable hexadecimal format is the same as the printable except all raw
+byte array elements will be printed as in the hexadecimal.
```

Comment:
Change `will be printed as in the hexadecimal` to `will be printed in hexadecimal format.`